Graviton_vs_x86_monitoring_differences.md

```diff
@@ -0,0 +1,42 @@
+# x86 monitoring differences
+## Utilities for obtaining processor stats
+|Processor	|x86	|Graviton2,3	|
+|---	|---	|---	|
+|CPU frequency listing	|*lscpu, /proc/cpuinfo, dmidecode*	|*dmidecode*	|
+|*turbostat* support	|Yes	|No	|
+|*i7z* Works	|Yes	|No	|
+|Performance monitoring tools	|_[VTune Profiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html) and [PCM](https://github.com/opcm/pcm), [Linux perf](https://www.brendangregg.com/perf.html)_	|_[Linux perf](https://www.brendangregg.com/perf.html), [Arm Forge](https://developer.arm.com/Tools%20and%20Software/Arm%20Forge)_	|
+
+More details on how to use Linux perf utility on AWS Graviton processors is available [here](https://github.com/aws/aws-graviton-getting-started/blob/main/optimizing.md#profiling-the-code).
+
+**Note:** On AWS Graviton 2/3 processors, Turbo isn’t supported. So, utilities such as ‘turbostat’ aren’t supported on Arm architecture (and not on AWS Graviton processor either) as this tool is written for/ by Intel. Also, tools such as *[i7z](https://code.google.com/archive/p/i7z/)* for discovering CPU frequency, turbo, sockets and other information are only supported on Intel architecture/ processors.
+
+## Sample output
```

Comment:
we should talk about how sys-stat monitoring does do the same thing

Reply:
I will expand on this page and add sysstat. For now I didn't add yet.